### PR TITLE
Add SSM region and update from skeleton

### PR DESCRIPTION
**Pull request by dav3r**
# 

## 🗣 Description
This PR is mainly to add a newly-required `region` parameter to the `aws_ssm` lookup (see https://github.com/cisagov/ansible-manage-user/commit/5d1a69e4206a7fc6fb13bd19564b8f4c210fd60c).  I also took this opportunity to pull in most (but not all) of the goodies from [`skeleton-ansible-role`](https://github.com/cisagov/skeleton-ansible-role), which was not used when this project was first created.

## 💭 Motivation and Context
Sometime between the last time I ran these playbooks and now, the `region` parameter became required in the `aws_ssm` lookup, so this PR takes care of that.

## 🧪 Testing
All pre-commit checks pass and I manually validated that the playbooks could be successfully executed.

## 🚥 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+name: build
+
+on:
+  push:
+  pull_request:
+  repository_dispatch:
+    types: [apb]
```
Comment (felddy) [inline .github/workflows/build.yml:8]: 🎉

**Review by felddy — APPROVED** [review]
That's a lot of nuts!  🔩 🥜

File: meta/main.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        # does not exist in Fedora 30.  Until that is resolved, we
+        # can't use Fedora 30.
+        # - 30
+        - 31
+    - name: Ubuntu
+      versions:
+        - bionic
+        - xenial
+  galaxy_tags:
+    - skeleton
```
Comment (jsf9k) [inline meta/main.yml]: Either add tags that are appropriate or change this to `galaxy_tags: []`.

**Review by jsf9k — APPROVED** [review]
Looks good!  I requested one minor change.

File: meta/main.yml (hunk 2 above)
Comment (dav3r) [reply meta/main.yml]: Thanks for catching that! I added some tags in https://github.com/cisagov/ansible-manage-user/pull/4/commits/12da377215686287590d088e15d02ed82db50281.

**Pull request merged**